server/Dockerfile

```diff
@@ -104,13 +104,22 @@ RUN ./configure-apt.sh && \
         mesa-vulkan-drivers \
         tini \
         wget \
-        zlib1g && \
-    ./install-ffmpeg.sh && \
-    apt-get install -t unstable --no-install-recommends -yqq \
-        $(if [ $(arch) = "x86_64" ]; then echo "intel-media-va-driver-non-free intel-opencl-icd"; fi) \
-        libwebp7 \
-        libwebpdemux2 \
-        libwebpmux3 && \
+        zlib1g \
+        ocl-icd-libopencl1 && \
+    ./install-ffmpeg.sh
+
+    RUN apt-get install -t testing --no-install-recommends -yqq libwebp7 libwebpdemux2 libwebpmux3
+
+    RUN if [ $(arch) = "x86_64" ]; then \
+      wget https://github.com/intel/intel-graphics-compiler/releases/download/igc-1.0.17193.4/intel-igc-core_1.0.17193.4_amd64.deb && \
+      wget https://github.com/intel/intel-graphics-compiler/releases/download/igc-1.0.17193.4/intel-igc-opencl_1.0.17193.4_amd64.deb && \
+      wget https://github.com/intel/compute-runtime/releases/download/24.26.30049.6/intel-opencl-icd-dbgsym_24.26.30049.6_amd64.ddeb && \
+      wget https://github.com/intel/compute-runtime/releases/download/24.26.30049.6/intel-opencl-icd_24.26.30049.6_amd64.deb && \
+      wget https://github.com/intel/compute-runtime/releases/download/24.26.30049.6/libigdgmm12_22.3.20_amd64.deb && \
+      dpkg -i *.deb && \
+      apt-get install -t testing --no-install-recommends -yqq intel-media-va-driver-non-free && \
+      rm *.deb; \
+    fi && \
```

Comment:
I don't really like it, it's hard to tell if there are any missing dependencies.

I have something like this for the AIO image, which comes from linuxserver:
```bash
  if [ $(arch) = "x86_64" ]; then \
    echo "**** install intel dependencies ****" && \
    apt-get install --no-install-recommends -y \
      intel-media-va-driver-non-free \
      ocl-icd-libopencl1 && \
    INTEL_DEPENDENCIES=$(curl -sX GET "https://api.github.com/repos/intel/compute-runtime/releases/latest" | jq -r '.body' | grep wget | grep -v .sum | grep -v .ddeb | sed 's|wget ||g') && \
    mkdir -p /tmp/intel && \
    for i in $INTEL_DEPENDENCIES; do \
      curl -fS --retry 3 --retry-connrefused -o \
        /tmp/intel/$(basename "${i%$'\r'}") -L \
        "${i%$'\r'}"; \
    done && \
    dpkg -i /tmp/intel/*.deb; \
  fi && \
```

Not sure if we need `wget https://github.com/intel/compute-runtime/releases/download/24.26.30049.6/intel-opencl-icd-dbgsym_24.26.30049.6_amd64.ddeb`, it's not installed by `dpkg -i *.deb`

Reply:
Interesting! I guess an advantage of this is that you can just point it at a release (or latest) without specifying the specific file name of each dependency.

Reply:
I think the issue is we don't want all of the dependencies from the release, maybe we could filter which ones we take though

Reply:
Removed the unused ddeb package, leaving the rest as-is for now